modules/apache-airflow/templates/values.yaml

```diff
@@ -108,7 +108,9 @@ images:
     pullPolicy: IfNotPresent
 
 # Select certain nodes for airflow pods.
-nodeSelector: {}
+nodeSelector: {
+  failure-domain.beta.kubernetes.io/zone: us-east-1a
+}
```

Comment:
Note the changes in this values.yaml file to update replicas from 2 to 1. Running these services with on-demand instances will get us towards the stability we wanted. Running these workloads on spot instances was not the right decision.